start-it-with-local-env.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+#
+# SPDX-FileCopyrightText: 2024 Lifely
+# SPDX-License-Identifier: EUPL-1.2+
+#
+
+set -e
+
+help() {
+   echo "Starts the Integration Tests with local docker image."
+   echo
+   echo "Syntax: $0 [-l|h]"
+   echo "options:"
+   echo "-l     Build a local ZAC Docker image."
+   echo "-h     Print this Help."
+   echo
+}
+
+while getopts ':lh' OPTION; do
+  case $OPTION in
+    h)
+      help
+      exit;;
+    l)
+      echo "Building ZAC Docker Image ..."
+      ./gradlew buildDockerImage
+      ;;
+    \?)
+      echo "Error: Invalid option"
+      help
+      exit;;
+  esac
+done
+
+if sudo lsof -i:8080; then
+  echo "Please stop the currently running WildFly!"
+  exit 1
+fi
+
+VOLUMES_DIR=scripts/docker-compose/volume-data
```

Comment:
Is this really needed? I guess this only applies to you @hsiliev running on Linux..? ;-) I think it's a bit overkill and adds extra complexity?